### PR TITLE
rename label of 'Active' column to 'Status'

### PR DESCRIPTION
**Pull request by cfoellmann**

### CODE REVIEW

**Comment by scarstens** [comment]
+1 lets get this merged in too!

**Comment by johnjamesjacoby** [comment]
The database should probably mirror this too, before it gets too late.

The `active` column takes after WordPress core's bad naming conventions when it comes to what a "status" means, is, and does.

**Comment by robrecord** [comment]
Could this be merged?

File: inc/admin/class-alias-list-table.php [hunk 1]
```diff
@@ -237,8 +237,8 @@ protected function column_domain( $mapping ) {
 	protected function column_active( $mapping ) {
 		$active = $mapping->is_active();
 		if ( $active ) {
-			return esc_html__( 'Active', 'mercator' );
+			return '<span style="color:green;font-weight:bold;">' . esc_html__( 'Active', 'mercator' ) . '</span>';
```
Comment (roborourke) [inline inc/admin/class-alias-list-table.php]: Yo, not sure the bold is necessary, also can you use the same colours as the approve/unapprove links from the comments area? Pity those class names are `.approve` and `.unapprove`. `#006505` for active and `#d98500` for inactive. Sorry for the lack of activity here for a while.

Comment (cfoellmann) [reply inc/admin/class-alias-list-table.php]: the css classes are not applicable in this context

**Pull request closed without merging**